### PR TITLE
Fix the description of the display property according to the CSS specification

### PR DESCRIPTION
**Pull request by oti**
### Description

"block element" and "inline element" term are old of HTML 4.01 era. The CSS Specification says "block level" and "inline level".

### Motivation

The incorrect keywords "block element" and "inline element" have been translated into Japanese as they are, causing problems that some web developers cannot understand CSS specifications correctly.

e.g. https://zenn.dev/coedo/articles/html-css-inline-element-block-level-element

### Additional details

https://www.w3.org/TR/css-display-3/#display-value-summary

### CODE REVIEW

File: files/en-us/web/css/display/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ browser-compat: css.properties.display
 
 {{CSSRef}}
 
-The **`display`** [CSS](/en-US/docs/Web/CSS) property sets whether an element is treated as a [block or inline element](/en-US/docs/Web/CSS/CSS_Flow_Layout) and the layout used for its children, such as [flow layout](/en-US/docs/Web/CSS/CSS_Flow_Layout), [grid](/en-US/docs/Web/CSS/CSS_Grid_Layout) or [flex](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout).
+The **`display`** [CSS](/en-US/docs/Web/CSS) property sets whether an element is treated as a [block or inline level](/en-US/docs/Web/CSS/CSS_Flow_Layout) and the layout used for its children, such as [flow layout](/en-US/docs/Web/CSS/CSS_Flow_Layout), [grid](/en-US/docs/Web/CSS/CSS_Grid_Layout) or [flex](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout).
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:10]:
```suggestion
The **`display`** [CSS](/en-US/docs/Web/CSS) property sets whether an element is treated as a [block or inline box](/en-US/docs/Web/CSS/CSS_Flow_Layout) and the layout used for its children, such as [flow layout](/en-US/docs/Web/CSS/CSS_Flow_Layout), [grid](/en-US/docs/Web/CSS/CSS_Grid_Layout) or [flex](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout).
```

File: files/en-us/web/css/display/index.md [hunk 2]
```diff
@@ -66,9 +66,9 @@ The keyword values can be grouped into six value categories.
   - : These keywords specify the element's outer display type, which is essentially its role in flow layout:
 
     - `block`
-      - : The element generates a block element box, generating line breaks both before and after the element when in the normal flow.
+      - : The element generates a block level box, generating line breaks both before and after the element when in the normal flow.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:69]:
```suggestion
      - : The element generates a block box, generating line breaks both before and after the element when in the normal flow.
```

File: files/en-us/web/css/display/index.md [hunk 3]
```diff
@@ -66,9 +66,9 @@ The keyword values can be grouped into six value categories.
...
     - `inline`
-      - : The element generates one or more inline element boxes that do not generate line breaks before or after themselves. In normal flow, the next element will be on the same line if there is space.
+      - : The element generates one or more inline level boxes that do not generate line breaks before or after themselves. In normal flow, the next element will be on the same line if there is space.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:71]:
```suggestion
      - : The element generates one or more inline boxes that do not generate line breaks before or after themselves. In normal flow, the next element will be on the same line if there is space.
```

File: files/en-us/web/css/display/index.md [hunk 4]
```diff
@@ -88,15 +88,15 @@ The keyword values can be grouped into six value categories.
         Depending on the value of other properties (such as {{CSSxRef("position")}}, {{CSSxRef("float")}}, or {{CSSxRef("overflow")}}) and whether it is itself participating in a block or inline formatting context, it either establishes a new [block formatting context](/en-US/docs/Web/Guide/CSS/Block_formatting_context) (BFC) for its contents or integrates its contents into its parent formatting context.
 
     - `flow-root`
-      - : The element generates a block element box that establishes a new [block formatting context](/en-US/docs/Web/Guide/CSS/Block_formatting_context), defining where the formatting root lies.
+      - : The element generates a block level box that establishes a new [block formatting context](/en-US/docs/Web/Guide/CSS/Block_formatting_context), defining where the formatting root lies.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:91]:
```suggestion
      - : The element generates a block box that establishes a new [block formatting context](/en-US/docs/Web/Guide/CSS/Block_formatting_context), defining where the formatting root lies.
```

File: files/en-us/web/css/display/index.md [hunk 5]
```diff
@@ -88,15 +88,15 @@ The keyword values can be grouped into six value categories.
...
     - `table`
       - : These elements behave like HTML {{HTMLElement("table")}} elements. It defines a block-level box.
     - `flex`
-      - : The element behaves like a block element and lays out its content according to the [flexbox model](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout).
+      - : The element behaves like a block level and lays out its content according to the [flexbox model](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout).
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:95]:
```suggestion
      - : The element behaves like a block-level element and lays out its content according to the [flexbox model](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout).
```

File: files/en-us/web/css/display/index.md [hunk 6]
```diff
@@ -88,15 +88,15 @@ The keyword values can be grouped into six value categories.
...
     - `grid`
-      - : The element behaves like a block element and lays out its content according to the [grid model](/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout).
+      - : The element behaves like a block level and lays out its content according to the [grid model](/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout).
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:97]:
```suggestion
      - : The element behaves like a block-level element and lays out its content according to the [grid model](/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout).
```

File: files/en-us/web/css/display/index.md [hunk 7]
```diff
@@ -88,15 +88,15 @@ The keyword values can be grouped into six value categories.
...
     - `ruby` {{Experimental_Inline}}
-      - : The element behaves like an inline element and lays out its content according to the ruby formatting model. It behaves like the corresponding HTML {{HTMLElement("ruby")}} elements.
+      - : The element behaves like an inline level and lays out its content according to the ruby formatting model. It behaves like the corresponding HTML {{HTMLElement("ruby")}} elements.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:99]:
```suggestion
      - : The element behaves like an inline-level element and lays out its content according to the ruby formatting model. It behaves like the corresponding HTML {{HTMLElement("ruby")}} elements.
```

File: files/en-us/web/css/display/index.md [hunk 8]
```diff
@@ -180,13 +180,13 @@ This can be used together with {{CSSxRef("list-style-type")}} and {{CSSxRef("lis
 
     - `inline-flex`
 
-      - : The element behaves like an inline element and lays out its content according to the flexbox model.
+      - : The element behaves like an inline level and lays out its content according to the flexbox model.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:183]:
```suggestion
      - : The element behaves like an inline-level element and lays out its content according to the flexbox model.
```

File: files/en-us/web/css/display/index.md [hunk 9]
```diff
@@ -180,13 +180,13 @@ This can be used together with {{CSSxRef("list-style-type")}} and {{CSSxRef("lis
...
 
         It is equivalent to `inline flex`.
 
     - `inline-grid`
 
-      - : The element behaves like an inline element and lays out its content according to the grid model.
+      - : The element behaves like an inline level and lays out its content according to the grid model.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:189]:
```suggestion
      - : The element behaves like an inline-level element and lays out its content according to the grid model.
```

**Review by estelle — COMMENTED** [review]
Thank you for your PR. 

You're correct that inline element and block element are no longer used. Thanks for the PR. 

An element generates either a _block box_ or an _inline box_ (or one or more inline boxes if it goes across multiple lines). The element, therefore, is a _block-level element_ or an _inline-level element_. I made some suggestions to reflect this.

Welcome to the team!

**Comment by oti** [comment]
@estelle I fixed it as per your suggestion. Would you please review.

**Comment by oti** [comment]
I'm glad to see that the other "inline element/block element" descriptions on MDN have been corrected! #25891

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to the team!

**Pull request merged**